site/documentation/content/concepts/resource-limits.md

```diff
@@ -14,6 +14,42 @@ Before accepting a new connection request from a device, the number of existing
 
 The MQTT and AMQP protocol adapters keep the connections longer opened than their counterparts such as HTTP. Thereby the MQTT and AMQP adapters are enabled to check the connection limits before accepting any new connection to a device.
 
+## Connection Duration Limit
+
+Before accepting a new connection request from a device, the overall amount of time that the devices been already connected to protocol adapters for that tenant is checked against the configured limit by the protocol adapters. The connection request is declined if the connection duration limit has been already reached. This limit is only supported by protocol adapters that maintain *connection state* with authenticated devices. In particular, the HTTP adapter does not support this metric.
+
+The default Prometheus based implementation uses connection duration as the factor to limit the connections. This default implementation supports two modes of connection duration limit calculation namely `days` and `monthly`. For more details on how to set the mode refer to the [Tenant API]({{< relref "/api/tenant#resource-limits-period-configuration-format" >}}).
+
+In the `monthly` mode, further device connections are only allowed, if the overall amount of time that the devices been already connected from the beginning till the end of a (Gregorian) calendar month does not exceed the configured *max-minutes* value. But for the first month, on which the connection duration limit became effective, the *effective connection duration limit* is calculated based on the *max-minutes* with respect to the remaining days in that month from the *effective-since* date.
```

Comment:
... that the devices *have* already been connected from the beginning till the end of *the current* (Gregorian) calendar month ...